templates/swift/Sources/Services/Realtime.swift.twig

```diff
@@ -20,6 +23,33 @@ open class Realtime : Service {
     private var subscriptionsCounter = 0
     private var reconnect = true
 
+    private func startHeartbeat() {
+        stopHeartbeat()
+        heartbeatTask = Task {
+            do {
+                while !Task.isCancelled {
+                    if let client = socketClient, client.isConnected {
+                        let pingMessage = ["type": "ping"]
+                        let jsonData = try JSONSerialization.data(withJSONObject: pingMessage)
+                        if let jsonString = String(data: jsonData, encoding: .utf8) {
+                            client.send(text: jsonString)
+                        }
```

Comment:
Might as well just send the raw JSON here to avoid the serialization overhead

```suggestion
                        client.send(text: #"{"type": "ping"}"#)
```